README_RISE.md

```diff
@@ -0,0 +1,55 @@
+#####Instrukcja instalacji wtyczki RISE służącej do obsługi interaktywnego pokazu slajdów
+
+
+# UNIX 
+
+1. Pobierz potrzebne pliki z repozytorium https://github.com/damianavila/RISE.git
+```
+cd
+git clone https://github.com/damianavila/RISE.git
+```
+
+2. Do instalacji należy wykorzystać skrypt `setup.py`, jednak wcześniej trzeba ustawić zmienną<br>
```

Comment:
Ustawianie czegokolwiek w .bashrc jest uciazliwe dla uzytkownika (moze uzywac innej powloki, moze miec juz taka zmienna ustawiona, moze to zaburzac inne ustawienia w tym pliku). Jesli instalacje da sie przeprowadzic bez modyfikacji .bashrc to byloby swietnie.